data/human/post-war news.txt

```diff
@@ -0,0 +1,647 @@
+		"My invisible friend knows that"
+
+phrase "paranoia: people in power are aliens"
+	phrase
+		"somebody suspects that"
+	word
+		" "
+	word
+		"the politicians"
+		"government officials"
+		"members of Parliament"
+		"people on the Paradise worlds"
+		"people on the rich worlds"
+		"those weird folks from the Deep"
+		"corporate executives"
+		"the Republic's intelligence operatives"
+		"bank managers"
+		"Syndicate executives"
+		"most planetary governors"
+		"the executives of the Tarazed corporation"
+		"the executives of the Lionheart corporation"
+		"the executives of the Megaparsec corporation"
+		"the executives of the Lovelace corporation"
+	word
+		" are actually aliens."
+		" are aliens in disguise."
+		" are just pretending to be human."
+		" are aliens."
+
+phrase "paranoia: band members are aliens"
+	word
+		"I watched the video of"
+		"I listened to"
+		"I just got tickets to see"
+		"I just got back from seeing"
+		"I'm just coming back from seeing"
+		"I just heard"
+		"I just saw"
+		"I had a backstage pass to"
+	word
+		" "
+	phrase
+		"band"
+	word
+		" "
+	word
+		"playing a concert"
+		"doing a show"
+		"in concert"
+		"at their concert"
+		"playing a benefit concert"
+		"live"
+		"on tour"
+		"at a live concert"
+		"playing a gig"
+		"live in concert"
+	word
+		" at Tarazed Stadium"
+		" at the amphitheater"
+		" at a concert hall"
+		" in a nightclub"
+		" at Betelgeuse Stadium"
+		" at Lionheart Arena"
+		" at the Grange"
+		" at a resort"
+		" at Canyonland Park"
+		" in a stylin' party-hall"
+		""
+	word
+		", but after what I heard"
+		", but after what I've seen"
+		", and now that I've really looked at them"
+	word
+		" I'm sure they're"
+		" I know they're"
+		" it's clear to me now they're"
+		" I know they have to be"
+	word
+		" aliens"
+	word
+		" in disguise."
+		" concealed by a hologram."
+		" hiding among us."
+		" pretending to be human."
+
+phrase "paranoia: media creators are aliens"
+	word
+		"I checked out the latest installment of "
+		"I got hooked on "
+		"I was catching up on "
+		"I went to a fan convention for "
+	word
+		"'"
+	phrase
+		"media name"
+	word
+		"'"
+	word
+		" but now I'm sure that"
+		" but I think"
+		" and I know now that"
+	word
+		" one of the actors is"
+		" the director is"
+		" the lead actor is"
+		" the set designer is"
+		" the producer is"
+	word
+		" an alien"
+		" an alien"
+		" some kind of alien"
+	word
+		" in disguise."
+		" wearing holograms."
+		" hiding among us."
+		" pretending to be human."
+
+phrase "paranoia: people in my life are aliens"
+	phrase
+		"My (person in my life) must be"
+		"The (person in my life) near me must be"
+	word
+		" an alien because "
+	phrase
+		"(reason why someone is an alien)"
+	word
+		"."
+
+phrase "(reason why someone is an alien)"
+	word
+		"someone saw them"
+		"I saw them"
+	word
+		" sneeze non-stop for an hour"
+		" wearing dark sunglasses in a movie theater"
+		" eating a piece of metal"
+		" speaking in their dreams"
```

Comment:
Can't exactly imagine why someone would see most of these people asleep, but it's funny to think about so I say leave it in.